tests/slow/src/test/scala/tests/maven/MavenLspSuite.scala

```diff
@@ -128,49 +127,51 @@ object MavenLspSuite extends BaseImportSuite("maven-import") {
     } yield ()
   }
 
-  testAsync("error") {
-    cleanWorkspace()
-    val badPom = defaultPom.replace(
-      "<!--DEPENDENCY-->",
-      s"""
-         |<dependency> 
-         |  <groupId>non.existent</groupId>
-         |  <artifactId>bad</artifactId> 
-         |  <version>0.1.4</version>
-         |</dependency>
-         |""".stripMargin
-    )
-    for {
-      _ <- server.initialize(
-        s"""|/pom.xml
-            |$badPom
-            |""".stripMargin,
-        expectError = true
-      )
-      _ = assertNoDiff(
-        client.workspaceMessageRequests,
-        List(
-          importBuildMessage,
-          progressMessage
-        ).mkString("\n")
-      )
-      _ = assertNoDiff(
-        client.workspaceShowMessages,
-        ImportProjectFailed.getMessage
-      )
-      _ = assertStatus(!_.isInstalled)
-      _ = client.messageRequests.clear()
-      _ <- server.didSave("pom.xml")(_ => defaultPom)
-      _ = assertNoDiff(
-        client.workspaceMessageRequests,
-        List(
-          importBuildMessage,
-          progressMessage
-        ).mkString("\n")
-      )
-      _ = assertStatus(_.isInstalled)
-    } yield ()
-  }
+  // TODO(olafur) re-enable this test when it's no longer flaky
```

Comment:
Why are we totally disabling this maven test? I haven't seen it fail previously. And it's not really related to the linked issue :thinking: